packages/sw-runtime-caching/src/lib/network-first.js

```diff
@@ -13,13 +13,19 @@
  limitations under the License.
 */
 
+import {CacheableResponse} from '../../../sw-cacheable-response/src/index';
 import Handler from './handler';
 import assert from '../../../../lib/assert';
 
 /**
  * An implementation of a [network first](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#network-falling-back-to-cache)
  * request strategy.
  *
+ * By default, [opaque responses](http://stackoverflow.com/q/39109789)
```

Comment:
Also worth say "By default, Network First will cache ...." default is a tad generic.